test/test_support.ts

```diff
@@ -405,7 +366,7 @@ export function compileWithTransfromer(
   const program = ts.createProgram(fileNames, compilerOptions, tsHost);
   expectDiagnosticsEmpty(ts.getPreEmitDiagnostics(program));
 
-  const rootModulePath = rootPath ? rootPath : getCommonParentDirectory(fileNames);
+  const rootModulePath = rootPath ? rootPath : path.dirname(fileNames[0]);
```

Comment:
Why is this needed?

Reply:
Because I want to run this test in google3, where we don't compile `main.ts` at all, plus this method is poorly thought out as you point out above, so we're better off not using it. Again, this is in a separate commit.